web/cypress/integration/classifications/classifications.spec.js

```diff
@@ -1,151 +1,126 @@
 context('TASKANA Classifications', () => {
   beforeEach(() => cy.loginAs('admin'));
 
-  it('should be able to visit Classifications and filter by manual', () => {
+  it('should be possible to edit the Service Level of a classification', () => {
     cy.visit(Cypress.env('appUrl') + Cypress.env('adminUrl') + '/classifications');
+    cy.wait(Cypress.env('pageReload'));
+    cy.contains(Cypress.env('testValueClassificationSelectionName')).click();
+
+    const editedValue = 'P99D';
+    cy.get('#classification-service-level').clear().type(editedValue);
+    cy.get('button').contains('Save').click();
+
+    cy.reloadPageWithWait();
+
+    cy.wait(Cypress.env('pageReload'));
+    cy.get('#classification-service-level').should('have.value', editedValue);
+  });
 
-    cy.get('#dropdown-classification-filter')
-      .click()
-      .then(() => {
-        cy.contains('MANUAL').click();
 
-        cy.get('tree-node-collection').find('tree-node').should('have.length', 2);
-      });
+  it('should be able to visit Classifications and filter by manual', () => {
+    cy.visit(Cypress.env('appUrl') + Cypress.env('adminUrl') + '/classifications');
+    cy.wait(Cypress.env('pageReload'));
+ 
+    cy.get('button[mattooltip="Filter Category"]').click()
+    .then(() => {
+      cy.get('.mat-menu-content').contains('MANUAL').click()
+      cy.get('tree-node-collection').find('tree-node').should('have.length', 2);
+    });
   });
 
+
   it('should be possible to edit the Name of a classification', () => {
     cy.visit(Cypress.env('appUrl') + Cypress.env('adminUrl') + '/classifications');
+    cy.wait(Cypress.env('pageReload'));
     cy.contains(Cypress.env('testValueClassificationSelectionName')).click();
 
     const editedValue = 'CY-TEST';
-
     cy.get('#classification-name').clear().type(editedValue);
+    cy.get('button').contains('Save').click();
 
-    cy.get('[title="Save"] > .material-icons').click();
-
-    cy.reload();
+    cy.reloadPageWithWait();
 
     cy.wait(Cypress.env('pageReload'));
-
     cy.get('#classification-name').should('have.value', editedValue);
   });
 
-  it('should be possible to edit the Priority of a classification', () => {
-    cy.visit(Cypress.env('appUrl') + Cypress.env('adminUrl') + '/classifications');
-    cy.contains(Cypress.env('testValueClassificationSelectionName')).click();
-    cy.get('input[name="number"]')
-      .invoke('val')
-      .then((oldPriorityValue) => {
-        oldPriorityValue = parseFloat(oldPriorityValue);
-        cy.get('[title="increase value"] > .material-icons').click();
-        cy.get('[title="Save"] > .material-icons').click();
-        cy.reload();
-        cy.wait(Cypress.env('pageReload'));
-        cy.get('.input-group > .form-control')
-          .invoke('val')
-          .then((newValueOfPriority) => {
-            newValueOfPriority = parseFloat(newValueOfPriority);
-            expect(newValueOfPriority).to.eq(oldPriorityValue + 1);
-          });
-      });
-  });
```

Comment:
I deleted this test because the selection of the 'increase value' field was not possible for me.
If this functionality is not included, the test does not appear to be useful.